walkthroughs/ec2/appmesh-proxy-route-manager.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash -e
```

Comment:
Is this changed from the script in the public docker image (111345817488.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager:v2)? I think it'd be better if we open source this image here/elsewhere rather than copying it inline here, as its useful for k8s as well.

Reply:
It is changed to make the variables overridable by the environment, and also AFAIK while the script is published in the docs, the source (nor Dockerfile) is in any public repo. I don't mind at all putting this into a separate project, but I'd like to do that as a change later and get this sample merged/published asap, if you're ok with that.